### PR TITLE
Fix(esmm_example):add one more classification task

### PR DESCRIPTION
**Pull request by Hjh233**
According to line110-111 in mtl_trainer.py:
if isinstance(self.model, ESMM):
                loss = sum(loss_list[1:])  #ESSM only compute loss for ctr and ctcvr task
we need to set three classification tasks in task_types to make sure total_loss = loss_{ctr} + loss_{ctcvr}, otherwise, total_loss = loss_{ctr}.

### CODE REVIEW

**Comment by yinpu** [comment]
模型这样修改后和esmm原始论文里面的实现不一样了
image
